### PR TITLE
Add player_onboarding 

### LINKED ISSUE #590: Nudge people towards more engagement
On this latest map so far:
266 players, and of them 175 never looked at more than just the first tab of the info screen (forced on join). A further 22 looked at more than 1 but not all. And 69 looked at all 5 tabs.

Lin's thoughts via discord:
>Could probably do some sort of help messages over time :thinking:
Linaori: after 30 seconds
>- check if the player has said something
>- if not, tell them how to talk
after 60 seconds, point them to the help screen

I would increase the times by a factor of 20, but I like the idea of small nudges. One poke after 10 minutes of silence might be welcome. Similarly a last poke after 20 minutes or so of silence about the info screen seems a propos.

### LINKED ISSUE #736: Toast instructions on how to chat for new players
If a player has not said anything in chat for 5 minutes (exact time TBD) and is not a regular, they are given a toast with instructions on how to send messages in chat.

### PR DESCRIPTION
**Pull request by mheguy**
Closes #590
Closes #736

### CODE REVIEW

File: features/teach_chat.lua [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        if player_chatted[player_index] then
+            return
+        end
+
+        local player = Game.get_player_by_index(player_index)
+        if not player or not player.valid or not player.connected then
+            return
+        end
+
+        Toast.toast_player(player, 30, {'teach_chat.chatting_text', {'gui-menu.settings'}, {'gui-menu.controls'}, {'controls.toggle-console'}})
```
Comment (linaori) [inline features/teach_chat.lua]: What if we also mention (in this toast): "You can close this toast message by clicking it, or waiting for the timer to run out", and change it to 60 seconds? Two birds, one stone :smile:

Comment (mheguy) [reply features/teach_chat.lua]: Love the overall idea, just tweaked the specifics.

**Comment by mheguy** [comment]
RB + FP for merge conflict. And an extra FP to squash the commits. :D

File: locale/en/redmew_features.cfg [hunk 2]
```diff
@@ -100,3 +100,7 @@ mention_success=__1__ __2__ mentioned __3__!
 mention_fail_mention_self=__1__ Can't mention yourself!
 mention_not_found_plural=__1__ Players not found: __2__
 mention_not_found_singular=__1__ Player not found: __2__
+
+[player_onboarding]
+teach_toast=This is a feature in RedMew called a toast!\nThey're little pieces of information that pop up.\nYou can dismiss is by waiting or by clicking on it.
```
Comment (SimonFlapse) [inline locale/en/redmew_features.cfg]:
```suggestion
teach_toast=This is a feature in RedMew called a toast!\nThey're little pieces of information that pop up.\nYou can dismiss it by waiting or by clicking on it.
```

**Review by linaori — APPROVED** [review]

**Pull request merged**